ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/BucketIntegration_Test.java

```diff
@@ -104,6 +107,33 @@ public void deleteBucket() throws IOException, SignatureException {
     }
 
     @Test
+    public void modifyJob() throws IOException, SignatureException, XmlProcessingException, URISyntaxException {
+        final String bucketName = "test_modify_job";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            List<Ds3Object> objects = new ArrayList<>();
+            final File objFile = ResourceUtils.loadFileResource("books/beowulf.txt");
```

Comment:
I don't think you actually have to load a real file for this test.  If you just create a fake `Ds3Object` that should work.